### ISSUE TITLE
Translate Polkadot A to Z: B for Bridges into Italian

### ISSUE THREAD

**Issue by emresurmeli** [w3f/1KC#15]
Original post:
https://www.reddit.com/r/Polkadot/comments/ocdnak/b_for_bridges_polkadot_from_a_to_z/

Task details:
To translate the Polkadot A to Z: A for Accounts blog post to Hausa. The reward will be calculated at 10 cents/word, for a total of 642 words equaling to a $64.20 reward. Payout format will be up to the Tech Ed team, in the form of a stable coin of their choice. The translation work will need to be QA'ed internally, the team may request changes from the original translator, and reserve the right to deny and work that does not meet minimum standards based on discretion of the reviewing team/s.

Task assigned to:
Patrizia | ANAMIX

Working document:
https://docs.google.com/document/d/1e5Cx91zrAjTLq6Z_ufwWr2areCuF4deYhFA7ZS6vF9M/edit

**Comment by dbpatty** [comment]
@emresurmeli Done.

**Comment by emresurmeli** [comment]
Thank you @dbpatty, @FedeC87p could you review when you get a chance?

**Comment by FedeC87p** [comment]
@emresurmeli I need the access to read the google document (I just sent the request)

**Comment by FedeC87p** [comment]
Hi @emresurmeli , I'm still waiting for the permissions on google document for validate the translation

**Comment by DrW3RK** [comment]
@dbpatty @FedeC87p Emre is on vacation till the end of the month. Let me know the action items pending on this issue. Will get them resolved.

**Comment by FedeC87p** [comment]
yes, thank. I need the read access to https://docs.google.com/document/d/1e5Cx91zrAjTLq6Z_ufwWr2areCuF4deYhFA7ZS6vF9M

**Comment by emresurmeli** [comment]
@FedeC87p Hi Fede, I'm back and I've given you access to the document.

**Comment by FedeC87p** [comment]
Hi @emresurmeli, I just finished the review and everything is OK. In my opinion the part where we talk about ChainX is outdated.

**Comment by emresurmeli** [comment]
@FedeC87p Thanks for the review, @dbpatty if we can fix the outdated section about ChainX that would be great!

**Comment by dbpatty** [comment]
> @FedeC87p Thanks for the review, @dbpatty if we can fix the outdated section about ChainX that would be great!

The Italian translation  is ok, I think that he suggests that the original text in English is outdated, do you want that I update them both?
